packages/ui/src/fields/Upload/Input.tsx

```diff
@@ -315,6 +322,35 @@ export function UploadInput(props: UploadInputProps) {
     [serverURL, api, code, i18n.language, t],
   )
 
+  const normalizeValue = useCallback(
```

Comment:
It is a bit hard to believe this isn't an exported function someplace else. Could we make this DRY?